app/pastes/get-paste/getPasteInteractor.js

```diff
@@ -26,6 +28,16 @@ class GetPasteInteractor {
             return;
         }
 
+        if (paste.getVisibility() === visibility.PUBLIC) {
```

Comment:
Lets make paste entity into true _entity_: provide it with `.isPublic` method, which would return boolean. From now we would try to avoid anemic models